cpp/log/etcd_consistent_store-inl.h

```diff
@@ -18,7 +18,7 @@ namespace {
 // etcd path constants.
 const char kClusterConfigFile[] = "/cluster_config";
 const char kUnsequencedDir[] = "/unsequenced/";
-const char kSequencedDir[] = "/sequenced/";
+const char kSequenceFile[] = "/sequencing";
```

Comment:
Not sure I find that name very enticing, but I haven't got much better... "mapping"?